tests/core/pyspec/eth2spec/test/deneb/sanity/test_blocks.py

```diff
@@ -58,3 +69,13 @@ def test_max_blobs_per_block(spec, state):
 @spec_state_test
 def test_invalid_exceed_max_blobs_per_block(spec, state):
     yield from run_block_with_blobs(spec, state, blob_count=spec.MAX_BLOBS_PER_BLOCK + 1, valid=False)
+
+
+@with_deneb_and_later
+@spec_state_test
+def test_some_blobs_across_several_blocks(spec, state):
+    block_count = 4
+    seed = 69420
+
+    random.seed(seed)
+    yield from run_block_with_blobs(spec, state, block_count=block_count)
```

Comment:
I refactored `run_block_with_blobs` to produce more than one blob; so, for this test at least, we can have a string of blocks (and ensure the parent root via 4788 is working correctly)